src/flb_log_event_decoder.c

```diff
@@ -300,3 +300,51 @@ int flb_log_event_decoder_next(struct flb_log_event_decoder *context,
                                            event,
                                            &context->unpacked_event.data);
 }
+
+const char* flb_log_event_encoder_strerror(int result)
```

Comment:
The name of this function is inverted with the one from the encoder.

Reply:
Oops, I fixed it.